.github/workflows/CD.yml

```diff
@@ -150,6 +151,8 @@ jobs:
           pip install --upgrade pip
           pip install numpy==1.26.4
           pip install -r dependencies.txt
+          pip uninstall loopprojectfile -y
```

Comment:
This is just for the PR and will be removed when merging to master

Reply:
yep, there are notes at the bottom of the PR on what to change before merging into master. 
Will add them as tasks.